types/templates.go

```diff
@@ -346,11 +346,11 @@ type ValidatorPageData struct {
 	SlashedBy                                uint64
 	SlashedAt                                uint64
 	SlashedFor                               string
-	ActivationEligibilityEpoch               uint64         `db:"activationeligibilityepoch"`
-	ActivationEpoch                          uint64         `db:"activationepoch"`
-	ExitEpoch                                uint64         `db:"exitepoch"`
-	Index                                    uint64         `db:"index"`
-	LastAttestationSlot                      uint64         `db:"lastattestationslot"`
```

Comment:
I did not suggest to remove the `db` tag here. Now that you did, I checked again why I did not suggest it and I realized that I forgot to mention this in my previous review:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/4c13158c1df89dc46db427cdac0117c1ac58d446/services/rewards.go#L297-L301
This is still using `lastattestationslot` coming from Postgres. I guess this should be adapted so it takes the info from bigtable too? If so, removing the `db` tag from `LastAttestationSlot` in `ValidatorPageData` in fine.